connman/plugins/sailfish_ofono.c

```diff
@@ -662,26 +665,34 @@ static gboolean delayed_connect(gpointer data) {
 
 	connman_network_set_connected(md->network, TRUE);
 
-	g_source_remove(md->delayed_connect_id);
 	md->delayed_connect_id = 0;
 	md->delayed_connect_attempts = 0;
 
-	return FALSE;
+	return G_SOURCE_REMOVE;
+}
+
+static void clean_modem_delayed_connect(struct modem_data *md)
```

Comment:
I think clean_ is somewhat ambiguous here (at least I interpret clean primarily as an adjective, maybe I'm a lazy cleaner...), I'd prefer clear_ or cleanup_, but then again, naming of internal functions is not that important.

Reply:
Yeah maybe `modem_clear_delayed_connect()` sounds better anyways? Changed to use that.

Reply:
Renamed further to be in line with the rest.